module-gui/gui/widgets/Item.hpp

```diff
@@ -79,7 +79,7 @@ namespace gui
         BoundingBox drawArea; // drawableArea would be more accurate
                               // maximal bounding box size
         /// gets bounding box for selected area
-        auto area(Area which = Area::Normal) -> BoundingBox &
```

Comment:
just asking, why did you change the trailing return to a standard one?

Reply:
Because:
1. For consistency, the standard return type is used for majority of functions in this class.
2. Trailing return type is not required in this case
3. The line is shorter

I don't have a strong opinion about it though. If you prefer the trailing type I can revert it.

But I think that if this is prefered then it should probably be done for all of the functions in the codebase or at least in this file. This way all of the function names would be aligned and the return type would always be at the end. So a reader would know where to look in all of the places. The function names would be easier to read but return types especially for functions with long list of parameters would be harder to read. Depends what we prefer.